README.md

```diff
@@ -28,3 +28,50 @@ Open two shell/terminal sessions:
 _Refer to `manage -h` for additional usage information._
 
 Once services are started, visit [http://localhost:8080](http://localhost:8080) to see the app running and visit [http://localhost:8050](http://localhost:8050) to see any outbound mail the app is sending (it won't actually send any email message in the development environment.)
+
```

Comment:
Use this content to make updates to the [BCDevOps/openshift-developer-tools](https://github.com/BCDevOps/openshift-developer-tools/blob/master/bin/README.md) documentation, and then link to it.

Reply:
updated readme and linked to openshift-dev-tools documents